web/src/app/modules/shared/components/form-view-container/form-view-container.component.html

```diff
@@ -89,8 +89,9 @@ <h6 *ngIf="field.config.label" slot="label">{{ field.config.label }}</h6>
       <cds-select [style.width.%]="field.config.width">
         <label [for]="field.config.name">{{ field.config.label }}</label>
         <select
-          [formControlName]="field.config.name"
+          [formArrayName]="field.config.name"
           [attr.multiple]="field.config.configuration.multiple"
+          (change)="onSelect($event, field.config.name)"
         >
           <option
             *ngFor="let opt of fieldChoices(field); trackBy: trackByFn"
```

Comment:
Missing `[selected]="opt.checked"` here if we want default selection to work

Reply:
Good Catch! [selected]="opt.checked" does not work, I don't think this solution works, I think is better just to use the regular Angular way

Reply:
Actually it does work but is not reflected on the HTML 
![image](https://user-images.githubusercontent.com/6991432/125541556-355e7e3b-27d4-443e-abce-24ffa353beb6.png)